lib/retrieve_title.rb

```diff
@@ -5,8 +5,9 @@ module RetrieveTitle
 
   def self.crawl(url)
     fetch_title(url)
-  rescue Exception
-    # If there was a connection error, do nothing
+  rescue Exception => ex
+    raise if Rails.env.test?
+    Rails.logger.error(ex)
```

Comment:
Many specs were "passing" but they were not actually testing anything because of this `Exception` rescue 😭

Reply:
😭